source/assets/scripts/main.js

```diff
@@ -311,6 +322,8 @@ function editTasks(taskID) {
  * check number of tasks marked done
```

Comment:
We are essentially updating checkbox status of task and task count of tasks in this function whenever checkbox is clicked. Method header should sth more like a sentence rather some bullet points.